*** tests/A-01-01-M-ok.out	2021-10-22 10:19:18.899175945 +0100
--- tests/A-01-01-M-ok.outhyp	2021-10-22 10:58:23.605688781 +0100
***************
*** 47,53 ****
  0 - Sair
  Escolha uma opção: Gestão de Transacções
  1 - Visualizar
! 2 - Registar desagregação
  3 - Registar venda
  4 - Registar compra
  5 - Receber pagamento de venda
--- 47,53 ----
  0 - Sair
  Escolha uma opção: Gestão de Transacções
  1 - Visualizar
! 2 - Registar compra
  3 - Registar venda
  4 - Registar compra
  5 - Receber pagamento de venda
